config/stage.json

```diff
@@ -41,6 +41,7 @@
 		"google-my-business": true,
 		"happychat": true,
 		"help": true,
+		"hosting/github-integration": true,
```

Comment:
@mpkelly, Do we need to activate this feature on stage?

Reply:
What's the disadvantage? Maybe it'll throw a12s off when they see it, or when it doesn't work quite right?

Reply:
Yes, I assumed that 99% of the time, staging would be the same as production so HE could use it as a reference. When I saw the GitHub Connection card, I was excited to try it, and I thought we had released it, and I missed the announcement. 
For reference p1675933707264989-slack-C0347E545HR